engine/spec/components/base_spec.rb

```diff
@@ -1,50 +0,0 @@
-# frozen_string_literal: true
-
-RSpec.describe CitizensAdviceComponents::Base, type: :component do
```

Comment:
We were already using the base component spec to do a version of this. But testing the module directly in the new spec file is much clearer.